packages/server/graphql/mutations/helpers/generateGroups.ts

```diff
@@ -53,6 +57,10 @@ const generateGroups = async (
 
   const r = await getRethink()
   await r.table('NewMeeting').get(meetingId).update({autogroupReflectionGroups}).run()
+  const data = {meetingId}
+  const operationId = dataLoader.share()
+  const subOptions = {operationId, mutatorId: SERVER_ID}
```

Comment:
We need the `mutatorId` but don't have access to the context here. I'm using the `server_id` so the subscription is published to everyone, but I'm not sure if there's a better approach

Reply:
-1 I would prefer if we'd publish subscription updates in mutations as it's easier to find and reason about. However I found 1 example where we do it in a helper and there we pass the `subOptions` as parameter https://github.com/ParabolInc/parabol/blob/4c607c8481398bc54684bb36ab7f239d1b11eb90/packages/server/graphql/public/mutations/helpers/updateNotification.ts#L5

Reply:
@nickoferrall 
Even if we don't await publishing to the subscription, we can call it from the mutation
```ts
const updateGeneratedGroups = async () => {
  await generateGroups(...)
  publish(...)
}
// don't await because it takes some time
updateGeneratedGroups()
```
I think this would make the intend clearer.
Regardless, I think omitting the `mutatorId` is the right move given the comment
https://github.com/ParabolInc/parabol/blob/3b3d9dd8f6de6eb8bf66b8d37a4d53ca9c1fa882/packages/server/utils/publish.ts#L4